src/components/modeler/Modeler.vue

```diff
@@ -744,6 +754,10 @@ export default {
       store.commit('highlightNode', this.processNode);
     }, this);
 
+    this.paperManager.addEventHandler('element:pointerclick', () => {
+      this.blurFocusedScreenBuilderElement();
+    }, this);
```

Comment:
Since you're explicitly passing in `this` as the callback scope, this could be simplified using an arrow function and it should work the same:
```js
this.paperManager.addEventHandler('element:pointerclick', this.blurFocusedScreenBuilderElement, this);
```

Reply:
Good catch, improved and updated.